config/nrfconnect/chip-module/CMakeLists.txt

```diff
@@ -99,9 +99,7 @@ if (CONFIG_POSIX_API)
 endif()
 
 if (CONFIG_NORDIC_SECURITY_BACKEND)
-    # TODO: Remove when the missing dependency is fixed in nRF Connect SDK
-    zephyr_include_directories(${ZEPHYR_BASE}/../nrfxlib/crypto/nrf_cc310_platform/include)
-
+    zephyr_include_directories($<TARGET_PROPERTY:mbedtls_external,INTERFACE_INCLUDE_DIRECTORIES>)
```

Comment:
Isn't Zephyr already linking with `mbedtls_external` ?

Like a: `zephyr_link_libraries(mbedtls_external)`.
If that is the case, then this shouldn't be needed and we should find out why it's not working as expected.

If there is no `zephyr_link_libraries(mbedtls_external)` anywhere then this code looks good.

Reply:
Note, Zephyr is not linking to mbedtls_external as both `MBEDTLS_BUILTIN=n` and `MBEDTLS_LIBRARY=n` when using Nordic security backend.